src/helpers/url.ts

```diff
@@ -0,0 +1,35 @@
+import { flow } from './fp';
+
+type Query = {
+  [index: string]: string | number | boolean;
+};
```

Comment:
Could we replace all instances of this with `URLSearchParams`?

Reply:
Not quite, because URLSearchParams is not a plain object but a class instance. So we'd have to upgrade our `query` arguments created in our methods to actually be `new URLSearchParams(query)`. It's a bit too involved of a change so I'd rather leave that out for later.

Reply:
Yeah, I was suggesting using `URLSearchParams` everywhere, rather than try to maintain two representations of the same thing.

Reply:
I see. I've added it to https://github.com/unsplash/unsplash-js/issues/145 to tackle post-adding the response types.